docs/HowTos/GitAndGitHub/git_setup.md

```diff
@@ -0,0 +1,17 @@
+# Git setup
+
+Make sure you set up git on every machine (including servers such as HPCs for example) you work on.
+
+## Config name and email
+
+All commits on git have information about the author with the name and email address. Before any work with Git, you should configure them globally for each machine you use. See [the GitHub documentation][git-setup] to guide you through it.
```

Comment:
Is "for each machine you use" redundant if it is said in the introduction?

Reply:
It is but I don't think I mind the repetition. It's a reminder more than a redundancy for me.